content/en/docs/contribute/_index.md

```diff
@@ -4,28 +4,30 @@ title: Contribute to Kubernetes docs
 linktitle: Contribute
 main_menu: true
 weight: 80
+card:
+  name: contribute
+  weight: 10
+  title: Contributing Overview
 ---
 
 {{% capture overview %}}
 
-If you would like to help contribute to the Kubernetes documentation or website,
-we're happy to have your help! Anyone can contribute, whether you're new to the
-project or you've been around a long time, and whether you self-identify as a
-developer, an end user, or someone who just can't stand seeing typos.
+The Kubernetes website loves all contributions from new members and experienced ones! Compared to other parts of the Kubernetes project, documentation is an easy way to
```

Comment:
nit: The word `website` is too generic
nit: loves? How about welcomes?

Reply:
You might want to introduce briefly he concept of SIG Docs, the Kubernetes documentation and website here. The first time SIG Docs is defined is in the last section of the page.